src/renderers/opengl/app.rs

```diff
@@ -82,7 +82,14 @@ where
             }
         }
 
-        let display = unsafe { Display::new(window.raw_display_handle(), Egl)? };
+        let display = unsafe { Display::new(window.raw_display_handle(), 
+            #[cfg(target_os = "linux")]
+            display::DisplayApiPreference::Egl,
+            #[cfg(target_os = "macos")]
+            display::DisplayApiPreference::Cgl,
+            #[cfg(target_os = "windows")]
+            display::DisplayApiPreference::Wgl,
```

Comment:
On Windows, it's actually [DisplayApiPreference::Wgl([Option](https://doc.rust-lang.org/nightly/core/option/enum.Option.html)<[RawWindowHandle](https://docs.rs/raw-window-handle/0.5.0/i686-pc-windows-msvc/raw_window_handle/enum.RawWindowHandle.html)>)](https://docs.rs/glutin/latest/i686-pc-windows-msvc/glutin/display/enum.DisplayApiPreference.html#variant.Wgl)

We have access to the raw window handle, so please `Some(raw_window_handle)` them ^^

Reply:
Have force pushed fix for this